shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -11,6 +11,15 @@ import scala.util.chaining._
 
 abstract class JSBackend(allowUnresolvedSymbols: Bool) {
   def oldDefs: Bool
+
+  protected implicit class TermOps(term: Term) {
```

Comment:
Do we have a better name for this? :thinking:

Reply:
This name follows the standard pattern.

Reply:
`XOps` generally provides extension methods for `X`.